src/components/index.ts

```diff
@@ -3,3 +3,4 @@ export { Footer } from './Footer';
 export { Layout } from './Layout';
 export { Main } from './Main';
 export { Navigation } from './Navigation';
+export { What } from './What';
```

Comment:
The name of this component should be more specific, `What` looks very confusing here.

Reply:
The name is now updated to Information